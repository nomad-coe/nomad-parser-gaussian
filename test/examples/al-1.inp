%mem=24000000
#ROHF/6-311G* 

atom calc
casno=7429905
method=9
basis=7

0 2
Al


