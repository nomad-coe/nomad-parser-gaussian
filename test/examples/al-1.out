 Entering Gaussian System, Link 0=g98
 Initial command:
 /usr/local/apps/g98/g98/l1.exe /wrk/rdj3/Gau-12985.inp -scrdir=/wrk/rdj3/
 Entering Link 1 = /usr/local/apps/g98/g98/l1.exe PID=     12986.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998 Gaussian, Inc.
                  All Rights Reserved.
  
 This is part of the Gaussian(R) 98 program.  It is based on
 the Gaussian 94(TM) system (copyright 1995 Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992 Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990 Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988 Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986 Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under DFARS:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, duplication or disclosure by the US Government is subject
 to restrictions as set forth in subparagraph (c)(1)(ii) of the
 Rights in Technical Data and Computer Software clause at DFARS
 252.227-7013.
  
 Gaussian, Inc.
 Carnegie Office Park, Building 6, Pittsburgh, PA 15106 USA
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is subject
 to restrictions as set forth in subparagraph (c) of the
 Commercial Computer Software - Restricted Rights clause at FAR
 52.227-19.
  
 Gaussian, Inc.
 Carnegie Office Park, Building 6, Pittsburgh, PA 15106 USA
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 98, Revision A.4,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, V. G. Zakrzewski, J. A. Montgomery, Jr., 
 R. E. Stratmann, J. C. Burant, S. Dapprich, J. M. Millam, 
 A. D. Daniels, K. N. Kudin, M. C. Strain, O. Farkas, J. Tomasi, 
 V. Barone, M. Cossi, R. Cammi, B. Mennucci, C. Pomelli, C. Adamo, 
 S. Clifford, J. Ochterski, G. A. Petersson, P. Y. Ayala, Q. Cui, 
 K. Morokuma, D. K. Malick, A. D. Rabuck, K. Raghavachari, 
 J. B. Foresman, J. Cioslowski, J. V. Ortiz, B. B. Stefanov, G. Liu, 
 A. Liashenko, P. Piskorz, I. Komaromi, R. Gomperts, R. L. Martin, 
 D. J. Fox, T. Keith, M. A. Al-Laham, C. Y. Peng, A. Nanayakkara, 
 C. Gonzalez, M. Challacombe, P. M. W. Gill, B. Johnson, W. Chen, 
 M. W. Wong, J. L. Andres, C. Gonzalez, M. Head-Gordon, 
 E. S. Replogle, and J. A. Pople, 
 Gaussian, Inc., Pittsburgh PA, 1998.

 *********************************************
 Gaussian 98:  x86-Linux-G98RevA.4 11-Sep-1998
                  18-Jul-2000 
 *********************************************
 %mem=24000000
 -------------
 #ROHF/6-311G*
 -------------
 1/38=1/1;
 2/17=6,18=5/2;
 3/5=4,6=6,7=1,11=2,25=1,30=1/1,2,3;
 4/7=6/1;
 5/5=2,32=1,38=4/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 -----------------------
 atom calc casno=7429905
 -----------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 2
 Al
 ------------------------------------------------------------------------
                         Z-MATRIX (ANGSTROMS AND DEGREES)
 CD Cent Atom  N1     Length/X     N2    Alpha/Y     N3     Beta/Z      J
 ------------------------------------------------------------------------
   1   1  Al
 ------------------------------------------------------------------------
                         Z-Matrix orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1         13             0        0.000000    0.000000    0.000000
 ---------------------------------------------------------------------
 Symmetry turned off:
 Atomic calculation.
 Stoichiometry    Al(2)
 Framework group  KH  
 Deg. of freedom    0
 Full point group                 KH      NOp   1
 Standard basis: 6-311G(d) (5D, 7F)
 Integral buffers will be    262144 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned off.
    26 basis functions       46 primitive gaussians
     7 alpha electrons        6 beta electrons
       nuclear repulsion energy         0.0000000000 Hartrees.
 One-electron integrals computed using PRISM.
 NBasis=    26 RedAO= T  NBF=    26
 NBsUse=    26 1.00D-04 NBFU=    26
 Projected CNDO Guess.
 Requested convergence on RMS density matrix=1.00D-04 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-02.
 Requested convergence on             energy=5.00D-05.
 Keep R1 and R2 integrals in memory in canonical form, NReq=      580172.
 SCF Done:  E(ROHF) =  -241.869965540     A.U. after    6 cycles
             Convg  =    0.2715D-04             -V/T =  2.0002
             S**2   =   0.7500
 Annihilation of the first spin contaminant:
 S**2 before annihilation     0.7500,   after     0.7500

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Alpha  occ. eigenvalues --  -58.50156  -4.90822  -3.21733  -3.21733  -3.21385
 Alpha  occ. eigenvalues --   -0.38273  -0.04455
 Alpha virt. eigenvalues --    0.01981   0.01981   0.17803   0.19123   0.19123
 Alpha virt. eigenvalues --    0.26979   0.65742   0.65742   0.65877   0.65877
 Alpha virt. eigenvalues --    0.65935   0.84766   0.84766   0.85418   4.41849
 Alpha virt. eigenvalues --    8.63326   8.63326   8.63511 122.50603
          Condensed to atoms (all electrons):
              1
  1  Al  13.000000
 Total atomic charges:
              1
  1  Al   0.000000
 Sum of Mulliken charges=   0.00000
 Atomic charges with hydrogens summed into heavy atoms:
              1
  1  Al   0.000000
 Sum of Mulliken charges=   0.00000
          Atomic-Atomic Spin Densities.
              1
  1  Al   1.000000
 Total atomic spin densities:
              1
  1  Al   1.000000
 Sum of Mulliken spin densities=   1.00000
                       Isotropic Fermi Contact Couplings
     Atom                 a.u.       MegaHertz       Gauss      10(-4) cm-1
   1  Al(27)             0.00000       0.00000       0.00000       0.00000
 Electronic spatial extent (au):  <R**2>=    33.3402
 Charge=     0.0000 electrons
 Dipole moment (Debye):
    X=     0.0000    Y=     0.0000    Z=     0.0000  Tot=     0.0000
 Quadrupole moment (Debye-Ang):
   XX=   -19.9441   YY=   -12.4500   ZZ=   -12.4500
   XY=     0.0000   XZ=     0.0000   YZ=     0.0000
 Octapole moment (Debye-Ang**2):
  XXX=     0.0000  YYY=     0.0000  ZZZ=     0.0000  XYY=     0.0000
  XXY=     0.0000  XXZ=     0.0000  XZZ=     0.0000  YZZ=     0.0000
  YYZ=     0.0000  XYZ=     0.0000
 Hexadecapole moment (Debye-Ang**3):
 XXXX=   -73.0035 YYYY=   -27.6779 ZZZZ=   -27.6779 XXXY=     0.0000
 XXXZ=     0.0000 YYYX=     0.0000 YYYZ=     0.0000 ZZZX=     0.0000
 ZZZY=     0.0000 XXYY=   -16.7802 XXZZ=   -16.7802 YYZZ=    -9.2260
 XXYZ=     0.0000 YYXZ=     0.0000 ZZXY=     0.0000
 N-N= 0.000000000000D+00 E-N=-5.784395170191D+02  KE= 2.418204398905D+02
 1\1\GINC-QCHEM5\SP\ROHF\6-311G(d)\Al1(2)\RDJ3\18-Jul-2000\0\\#ROHF/6-3
 11G*\\atom calc casno=7429905\\0,2\Al\\Version=x86-Linux-G98RevA.4\HF=
 -241.8699655\RMSD=2.715e-05\Dipole=0.,0.,0.\PG=KH\\@


            THE IRISH PIG
 'TWAS AN EVENING IN NOVEMBER,
 AS I VERY WELL REMEMBER
 I WAS STROLLING DOWN THE STREET IN DRUNKEN PRIDE,
 BUT MY KNEES WERE ALL A'FLUTTER
 SO I LANDED IN THE GUTTER,
 AND A PIG CAME UP AND LAY DOWN BY MY SIDE.

 YES, I LAY THERE IN THE GUTTER
 THINKING THOUGHTS I COULD NOT UTTER
 WHEN A COLLEEN PASSING BY DID SOFTLY SAY,
 'YE CAN TELL A MAN THAT BOOZES
 BY THE COMPANY THAT HE CHOOSES.' -
 AT THAT, THE PIG GOT UP AND WALKED AWAY!
     -- THE ECONOMIST, AUGUST 23, 1986
 Job cpu time:  0 days  0 hours  0 minutes 21.3 seconds.
 File lengths (MBytes):  RWF=    6 Int=    0 D2E=    0 Chk=    5 Scr=    1
 Normal termination of Gaussian 98.
